### DISCUSSION TITLE
Update on the development branch

### DISCUSSION THREAD

**Discussion by kaiyux** [NVIDIA/TensorRT-LLM#756] [Announcements]
Hi,

The TensorRT-LLM team is pleased to announce that we are pushing an update to the [development](https://github.com/NVIDIA/TensorRT-LLM/tree/main) branch (and the [Triton backend](https://github.com/triton-inference-server/tensorrtllm_backend/tree/main)) this December 27th, 2023.

This update includes:

* Features
  - Baichuan 
    - P-tuning support
    - INT4-AWQ and INT4-GPTQ support
  - The batch manager can now return logits from context and generation phases
  - Support for combining `repetition_penalty` and `presence_penalty` #274
  - Support for `frequency_penalty` #275
* Bug fixes
  - Fix LLaMA with LoRA enabled build failure #673
  - Fix InternLM SmoothQuant build failure #705
  - Fix Bloom int8_kv_cache functionality  #741
* Documentation
  - Batch manager arguments documentation updates
  - Add v0.7 change log

Happy holidays and advanced wishes to a happy new year!

Thanks,
The TensorRT-LLM Engineering Team
